core/shared/src/main/scala/zio/package.scala

```diff
@@ -75,4 +81,86 @@ package object zio
     def unapply[A, B](ab: (A, B)): Some[(A, B)] =
       Some((ab._1, ab._2))
   }
+
+  // Duration
```

Comment:
What if we move this to another file that is extended by the package object? There is already a lot that is going on in the package object and would be nice to keep that to higher level alias for various things instead of all the details of the duration implementation.